mso/resource_mso_schema_template_filter_entry_test.go

```diff
@@ -56,42 +61,100 @@ func TestAccMSOSchemaTemplateFilterEntry_Update(t *testing.T) {
 
 func testAccCheckMSOTemplateFilterEntryConfig_basic(ethertype string) string {
 	return fmt.Sprintf(`
+
+	resource "mso_schema" "schema1" {
+		name          = "Schema2"
+		template_name = "Template1"
+		tenant_id     = "5fb5fed8520000452a9e8911"
```

Comment:
is it possible to replace the tenant_id? (tenant_id = "5fb5fed8520000452a9e8911" --> tenant_id = "tenant1")